core/src/main/java/com/netease/arctic/op/OverwriteBaseFiles.java

```diff
@@ -101,6 +106,11 @@ public OverwriteBaseFiles withTransactionId(Long transactionId) {
     return this;
   }
 
+  public OverwriteBaseFiles set(String property, String value) {
```

Comment:
The parent class PartitionTransactionOperation has the same method for add snapshot summary. Now the override will break original logic

Reply:
yes, i would drop it